### PR TITLE
Make UI configurations translatable

### PR DESCRIPTION
**Pull request by wborn**
Adds the default translation strings so the UI configuration can be translated with Crowdin.

Other fixes/improvements:

* Adds config for this to crowdin.yml
* Adds the i18n Maven plugin dependency so it can be more easily used
* Fixes a few encoding issues
* Uses the ConfigurableService annotation

### CODE REVIEW

File: pom.xml
Comment (wborn) [inline pom.xml:73]: I probably need to update this soon. 

```suggestion
    <ohc.version>3.3.0-SNAPSHOT</ohc.version>
```

Comment (ghys) [reply pom.xml:73]: I'm assuming you're planning a merge after the release then?

Comment (wborn) [reply pom.xml:73]: Yes I think this is something for OH 3.3.0.

**Review by ghys — APPROVED** [review]
LGTM - waiting for other reviewers to chime in to merge.

**Comment by wborn** [comment]
I know some @openhab/add-ons-maintainers who always help with translations and reviewing such PRs. 😉

**Comment by martinvw** [comment]
LGTM

**Review by cweitkamp — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**
